src/internal/proc.js

```diff
@@ -403,6 +403,13 @@ export default function proc(
     const cancelPromise = promise[CANCEL]
     if(typeof cancelPromise === 'function') {
       cb.cancel = cancelPromise
+    } else {
```

Comment:
Oh, I apparently changed it. Originally I was storing `promise.abort` as a local variable. Will change to else if.